src/pages/LeeSoo/Login/LeeSooLogin.js

```diff
@@ -1,7 +1,76 @@
+/*eslint-disable*/
+
 import React from 'react';
+import { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './LeeSooLogin.scss';
 
 const LeeSooLogin = props => {
-  return <div>Login</div>;
+  let [LoginId, LoginIdChange] = useState('');
+  let [LoginPw, LoginPwChange] = useState('');
+  let [BtnColor, BtnColorChange] = useState('#41699d');
+
+  let BtnLogin = (LoginId.includes('@') && LoginPw.length >= 5) === true;
+
+  function LoginCheck(id, pw) {
+    if (id.includes('@') && pw.length >= 5) {
+      return false;
+    }
+    return true;
+  }
+
+  const navigate = useNavigate();
+  const goMain = () => {
+    navigate('/LeeSoo-Main');
+  };
+
+  return (
+    <div className="box">
+      <header className="logo">
+        <p>Westagram</p>
+      </header>
+      <div className="insert-area">
+        <form action="">
+          <input
+            type="text"
+            placeholder="전화번호,사용자 이름 또는 이메일"
+            onChange={e => {
+              LoginIdChange(e.target.value);
+            }}
+          />
+          <br />
+          {LoginId}
+          <br />
+          <br />
+          <input
+            type="password"
+            placeholder="비밀번호"
+            onChange={e => {
+              LoginPwChange(e.target.value);
```

Comment:
```suggestion

const handleChange = (e)=>{LoginPwChange(e.target.value)}
            onChange={  handleChange}

```
이벤트를 핸들링해주는 별개의 함수가 필요할것같아요!